### PR TITLE
AB#6467 Update "SpeakersGrid" to "FeaturedSpeakers" component on home…

### PR DESCRIPTION
**Pull request by adoprog**
… page

### CODE REVIEW

**Review by jflheureux — CHANGES_REQUESTED** [review]

File: Website/src/items/content/EdgeWebsite/home/speakers/Page Components/Featured Speakers Section.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Fields:
+    - ID: "05607296-e17f-567f-898f-46bb9665d619"
+      Hint: title
+      Value: Featured Speakers
+    - ID: "25bed78c-4957-4165-998a-ca1b52f67497"
+      Hint: __Created
+      Value: 20210615T142052Z
+    - ID: "348777e9-0329-5c42-a43d-3db7d8a7a748"
+      Hint: content
+      Value: "Road-test the world's most trusted sports and fitness equipment – we'll be welcoming 2,000 brands at this year's PLAY! Summit."
```
[the remark this answers is not in the document]
Comment (adoprog) [reply Website/src/items/content/EdgeWebsite/home/speakers/Page Components/Featured Speakers Section.yml]: I'll rollback my change to /speakers, but this means the text will not be added/updated.

File: Website/src/items/content/EdgeWebsite/home/speakers/Page Components/Featured Speakers Section.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Template: "1f476f1c-822a-5a88-8f7e-2431b77d1b70"
+Path: /sitecore/content/EdgeWebsite/home/speakers/Page Components/Featured Speakers Section
+Languages:
+- Language: en
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "05607296-e17f-567f-898f-46bb9665d619"
+      Hint: title
+      Value: Featured Speakers
```
[the remark this answers is not in the document]
Comment (adoprog) [reply Website/src/items/content/EdgeWebsite/home/speakers/Page Components/Featured Speakers Section.yml]: I'll rollback my change to /speakers, but this means the text will not be added/updated.

File: Website/src/items/content/EdgeWebsite/home.yml [hunk 3]
```diff
@@ -75,10 +75,12 @@ SharedFields:
           uid="{E532127E-1D91-51EE-B517-B8CBBCC57A20}" />
         <r
           rname="SpeakersGrid"
```
[the remark this answers is not in the document]
Comment (adoprog) [reply Website/src/items/content/EdgeWebsite/home.yml]: Removed rname

**Review by scottmulligan — APPROVED** [review]
Looks good!

**Pull request merged**